etna/insights/migrations/0033_auto_20220304_2226.py

```diff
@@ -0,0 +1,411 @@
+# Generated by Django 3.2.12 on 2022-03-04 22:26
+
+
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    # We get the model from the versioned app registry;
+    # if we directly import it, it'll be the wrong version
+    tag = apps.get_model("taggit", "Tag")
```

Comment:
This tag model is actually used by everything... I would expect this set of tags to be added to their own model, keeping them separate from the others.

Reply:
Yes , Tags for the insights now have their own model InsightsTag. it looks like this now:

tag = apps.get_model("insights", "InsightsTag")